src/main/webapp/script.js

```diff
@@ -202,7 +203,7 @@ function getUrl() {
       selectedDepth = 20;
     }
   }
-  const url = `/data?depth=${selectedDepth}&mutationNum=${currMutationNum}&nodeName=${nodeName}`;
+  const url = `/data?depth=${selectedDepth}&mutationNum=${currMutationNum}&nodeName=${nodeName}&tokenName=${tokenName}`;
   return url;
```

Comment:
I know this is kind of addressed by Connor's PR but could we also highlight the nodes with the token? We could pass some version of roots again like we did before and highlight those nodes.

Reply:
i just change the color of those nodes! idk what's best for ui if you have other suggestions lmk lol